src/ui/views/ui_app.jade

```diff
@@ -27,7 +27,7 @@ block content
     h1
       strong Kibana
```

Comment:
Just realized there's a translatability problem here.  This is effectively `Kibana ${welcomeMessage}` which splits a sentence into fixed and dynamic parts.

Better would be: `${welcomeMessage}` where the message was `<strong>Kibana</strong> is loading. It's good stuff, so hang on.` etc.